homeassistant/const.py

```diff
@@ -21,6 +21,8 @@
 CONF_PLATFORM = "platform"
 CONF_HOST = "host"
 CONF_HOSTS = "hosts"
+CONF_PORT = "port"
+CONF_PORTS = "ports"
```

Comment:
Personally, I don't care for having a config entry called port and then one called ports that mean entirely different things. The names are too similar for how different they are. Maybe use plugs for what you are talking about? Or possibly outlets.